File: Gamestats/ClientApp/src/app/models/round.ts

import { Participant } from './participant';
import { Game } from './game';

export class Round implements IRound {
  roundNumber: number;
  roundId: number;
  //gameId: number;
  game: Game;
  participants: Participant[]

  public constructor() {
    this.participants = [];
  }
}

export interface IRound {
  roundNumber: number;
  roundId: number;
}

File: Gamestats/Models/Round.cs
using Microsoft.EntityFrameworkCore.Metadata.Internal;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Gamestats.Models
{
    public class Round
    {
        [Column(name: "round_number")]
        public int RoundNumber { get; set; }

        [Key]
        [Column(name: "round_id")]
        public int RoundId { get; set; }

        public virtual ICollection<Participant> Participants { get; set; }

        [Column(name: "game_id")]
        [ForeignKey("game_id")]
        public Game Game { get; set; }

    }
}
